runtime/ddr/module.xml

```diff
@@ -307,6 +307,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<library name="j9utilcore"/>
 			<library name="j9ddrautoblob"/>
 			<library name="j9gcautoblob"/>
+			<library name="omrsig">
```

Comment:
Can you explain why this is necessary? It's surprising to see this linked to the signal chaining library

Reply:
Removed.